### ISSUE TITLE
icesat2 hub not scaling up

### ISSUE THREAD

**Issue by jhamman** [pangeo-data/pangeo-cloud-federation#398]
Following some changes to the icesat2 profile list yesterday, we're having some problems getting user pods to schedule:

```
Event log
Server requested
2019-09-04 16:51:28+00:00 [Warning] 0/3 nodes are available: 3 node(s) didn't match node selector.
2019-09-04 16:51:38+00:00 [Normal] pod didn't trigger scale-up (it wouldn't fit if a new node is added)
Spawn failed: Timeout
```

@scottyhq - can you take a look at the logs to see if anything pops out at you? Are the pods in the profile list too big?

**Comment by scottyhq** [comment]
sure enough @jhamman, the icesat2 cluster needs some maintenance. the nodes have been up for 100+ days and the original configuration is for 20Gb disks. As far as I know Docker images remain if a node isn't shut down and our current configuration is messy, with 1 user-notebook node running all the time:

```
CLUSTER	NODEGROUP	CREATED			MIN SIZE	MAX SIZE	DESIRED CAPACITY	INSTANCE TYPE	IMAGE ID
pangeo	dask-worker	2019-04-09T16:30:58Z	0		20		0					
pangeo	ng-680864af	2019-03-28T23:07:24Z	3		3		3					
pangeo	test		2019-04-05T21:36:33Z	0		100		1					
pangeo	user-notebook	2019-04-09T16:38:17Z	0		20		0
```

autoscaler log:
```
Pod can't be scheduled on eksctl-pangeo-nodegroup-user-notebook-NodeGroup-XXFTQEPOCHSU, predicate failed: CheckNodeDiskPressure predicate mismatch, cannot put icesat2-staging/jupyter-scottyhq on template-node-for-eksctl-pangeo-nodegroup-user-notebook-NodeGroup-XXFTQEPOCHSU-2143810545988417440, reason: node(s) had disk pressure
```

I suspect the autoscaler sees there is already a node up and instead of launching a new one prefers to schedule there, but then runs into the disk pressure issue.

We should update the nodegroups to match the NASA deployment (I should be able to get to it this week)

**Comment by scottyhq** [comment]
Fixed by #401 and #402. We now use 100Gb EBS volumes on nodes and the icesat2 and nasa cluster configurations are the same.
